tests/yast2_gui/yast2_security.pm

```diff
@@ -60,9 +67,15 @@ sub run {
     send_key "alt-f";
     wait_screen_change { send_key "down" };
     wait_screen_change { send_key "alt-o" };
+    wait_still_screen 3;
+
+    reset_consoles;
+    wait_still_screen 10;
 
+    handle_logout;
```

Comment:
In VR code, you add 'script_run('free -g -h -t');', but in PR removed. Please confirmed before submit PR, and PR code should be same as VR.

Reply:
it's just check the fre memory. it's like debug infor. so remove it.